docs/pages/deploy-a-cluster/aws-gslb-proxy-peering-ha-deployment.mdx

```diff
@@ -0,0 +1,248 @@
+---
+title: "AWS Multi-Region High Availability Deployment Guide"
+description: "Deploying a high-availability Teleport cluster using Proxy Peering and Route 53 to create global server load balancing."
+---
+
+This deployment architecture features two important design decisions:
+
+1. AWS Route 53 latency-based routing is used for global server load balancing 
+   ([GSLB](https://www.cloudflare.com/learning/cdn/glossary/global-server-load-balancing-gslb/)).
+   This allows for efficient distribution of traffic across resources that are globally distributed.
+
+2. Teleport's [Proxy Peering](../architecture/proxy-peering.mdx) is used to reduce the total number of tunnel connections in the Teleport cluster.
+
+
+This deployment architecture isn't recommended for use cases where your users or resources are 
+clustered in a single region, or for Managed Service Providers needing to provide separate clusters 
+to customers.
+
+This architecture is best suited for globally distributed resources and end-users that prefer a single point of
+entry while also ensuring minimal latency when accessing connected resources.
+
+## Key deployment components
+- High-availability Teleport cluster
```

Comment:
What did you think of this comment? https://github.com/gravitational/teleport/pull/24164#discussion_r1189072435